bots-deployment-detector/src/utils.ts

```diff
@@ -0,0 +1,48 @@
+export const nethermindDeployerAddress = '0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8';
+export const fortaContractAddress = '0x61447385b019187daa48e91c55c02af1f1f3f863';
+
+export const eventAgentUpdated =
+  'event AgentUpdated(uint256 indexed agentId, address indexed by, string metadata, uint256[] chainIds)';
+
+export const eventAgentEnabled =
+  'event AgentEnabled(uint256 indexed agentId, bool indexed enabled, uint8 permission, bool value)';
+
+export const functionCreateAgent =
+  'function createAgent(uint256 agentId, address owner, string metadata, uint256[] chainIds) public';
+
+export interface FindingAgentInput {
+  name: string;
+  description: string;
+  alertId: string;
+}
+
+const createAgentInput: FindingAgentInput = {
+  name: 'Bot Deployment Detector',
+  description: 'Bot deployment detected',
+  alertId: 'FORTA-1',
+};
+
+const updateAgentInput: FindingAgentInput = {
+  name: 'Bot Updates Detector',
+  description: 'Bot update/upgrade detected',
+  alertId: 'FORTA-2',
+};
+
+const enableAgentInput: FindingAgentInput = {
+  name: 'Enable Bot Detector',
+  description: 'Detects whenever a bot is enabled',
```

Comment:
description should be explained what happened in the alert. so it should be similar to L21 & L27